proto/prysm/v1alpha1/generated.ssz.go

```diff
@@ -1,5 +1,5 @@
 // Code generated by fastssz. DO NOT EDIT.
-// Hash: da030c0ebc8b54fe7980b6bbe6d6795c74bab4f92ca977b2bc1c6d4fcf4ed25f
+// Hash: 4c414580a216e351fa6e30d490d439259cebef5fd122e765781b5ff0332ba6c9
```

Comment:
This file seemed to have been updated due to the inclusion of blinded Blob Sidecars

Reply:
> I originally had it but removed it. I think it's better to add those in Radek or your PR. It's more confusing to include them here

Reply:
I don't see blinded blobsidecar? Can you point me to the line?